narratives/ncov_sit-rep_2020-05-15.md

```diff
@@ -0,0 +1,220 @@
+<br><br>
+
+But, at this stage of the pandemic, genomic epidemiology is most urgently needed within hyperlocal communities through local public health offices.
+<br><br>
+
+As such, this will be our last weekly Situation Report for awhile as we shift our focus to supporting local public health officials.
+<br><br>
+
+We will, of course, continue daily updates to the global & regional builds as new sequences are posted to GISAID.
+When the big-picture data says something new, we'll also issue additional Situation Reports.
+All of these updates will be posted to the [Nextstrain twitter account](https://twitter.com/nextstrain) (as always).  
+<br><br>
+
+You can also find <a href="https://nextstrain.org/ncov-sit-reps/">all of our previous Situation Reports here</a>.
+
+```
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Viral mutations are normal](https://nextstrain.org/ncov/global/2020-05-14?c=num_date&d=tree,entropy&m=div&p=full&legend=open)
+
+SARS-CoV-2 is a [coronavirus](https://nextstrain.org/help/coronavirus/human-CoV). Like other coronaviruses, SARS-CoV-2 has a genome encoded in RNA (rather than DNA).
+<br><br>
+Viruses infect cells because they need to borrow cellular machinery to replicate. For RNA viruses, this process is quite error-prone, as most RNA polymerases (the cellular machines that make copies of RNA) aren't able to proofread and correct their work. This leads to frequent mutations in the viral genome; these are normal and expected.
+<br><br>
+Importantly, the vast majority of these mutations either "break" the virus such that it can't transmit and/or replicate anymore, or don't change the virus at all because of [redundant encoding](https://en.wikipedia.org/wiki/Synonymous_substitution).
+Rarely, a mutation can help the virus replicate and/or transmit better, but nearly all of these changes still have only a tiny effect.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [Viral mutations are useful for tracing outbreaks](https://nextstrain.org/ncov/global/2020-05-14?c=gt-ORF3a_57&d=tree,entropy&f_division=New%20York&m=div&p=full)
+
+These mutations can be used as "bread crumbs" to trace the history of a given sample. Just as the people in your family share unique combinations of genetic characteristics, closely related viral samples share unique combinations of genetic mutations.
+<br><br>
+For example, here we show the viral "family tree" with each sample colored by which amino acid is present at a specific location in the viral genome (ORF1a, site 57). Highlighting samples from New York, we see that most of the cases from New York and Europe have the amino acid Histidine ("H") rather than Glutamine ("Q") at this position. This, in combination with all the other sites in the genome, tells us that these cases are all closely related to each other.
```

Comment:
```suggestion
For example, here we show the viral "family tree" with each sample colored by which amino acid is present at a specific location in the viral genome (gene "ORF1a", site 57). 
Highlighting samples from New York, we see that most of the cases from New York and Europe have the amino acid Histidine ("H") rather than Glutamine ("Q") at this position. This, in combination with all the other sites in the genome, tells us that these cases are all closely related to each other.
<br><br>
You can also see the position in gene "ORF3a" highlighted on the chart at the bottom of the left.
```